hbase-server/src/test/java/org/apache/hadoop/hbase/replication/TestReplicationBase.java

```diff
@@ -204,6 +204,23 @@ static void configureClusters(HBaseTestingUtility util1,
     conf2.setBoolean("hbase.tests.use.shortcircuit.reads", false);
   }
 
+  static void configureClusters(HBaseTestingUtility util1,
```

Comment:
Should this be moved to TestNamespaceReplicationWithBulkLoadedData.java, as it's only used there? Or instead, use existing _configureClusters(HBaseTestingUtility util1, HBaseTestingUtility util2)_ and just do the extra config for the 3rd util there.